src/File.php

```diff
@@ -95,4 +95,21 @@ public static function isAbsolutePath(string $path): bool
 
         return $path[0] === '/';
     }
+
+    /**
+     * Return an absolute path (from the cwd) based on a given path. If this
+     * was already absolute, no changes are made.
+     *
+     * @param string $path
+     * @param string $cwd
+     * @return string
+     */
+    public static function makeAbsolutePath(string $path, string $cwd): string
```

Comment:
Have to make very sure that this value is never stored somewhere in a cache or javascript file due to build servers.

Reply:
In prod the cache is never used, so this is no issue. The only "optimization" check it does it when the target files is already there and it's newer than any of the sources. But this result it never cached.

Only in dev there might be absolute paths in the cache folder, but I see this as not so much an issue.